symbols/overlay29.yml

```diff
@@ -188,6 +188,18 @@ overlay29:
           2 means the current floor has a rescue point
         
         return: floor type
+    - name: TryForcedLoss
+      address:
+        NA: 0x22E0620
+        EU: 0x22E0F60
+        JP: 0x22E1CAC
+      description: |-
+        Attempts to trigger a forced loss of the type specified in the forced loss reason field of the dungeon struct. Returns true if the loss happens, false otherwise.
+        
+        If the parameter is true, the function will not check for the end of the floor condition and will skip other (unknown) actions in case of forced loss.
```

Comment:
Put this on the r0 line

Reply:
I don't think this was fixed

Reply:
Forgot about this too, I really need to make smaller commits